### DISCUSSION TITLE
Some questions about "layer"

### DISCUSSION THREAD

**Discussion by skyehan111** [openTCS/opentcs#3] [Q&A] [answered]
Hello, everyone

I tried to put multiple maps into a model. I noticed that there is a layer attribute in version 5.2. I don't know if this attribute can help me achieve the goal of creating multiple maps on a model. 

In addition, I want to know if there is an element of layer in version 5.0. I did not see any layer in the user manual of version 5.0.

Best regards
Skye

**Comment by swltr** [comment, marked as answer]
Hello Skye,

layers were introduced in openTCS 5.1. See the [release notes](https://www.opentcs.org/docs/5.2/release-notes.html) for changes between releases. How to work with layers is described in the [user's guide](https://www.opentcs.org/docs/5.2/user/opentcs-users-guide.html).

Whether this feature can help you depends on what you mean with "multiple maps". Even without the layers, you can have multiple separate graphs in a plant model. In case "map" refers to e.g. a [SLAM](https://en.wikipedia.org/wiki/Simultaneous_localization_and_mapping) map and you have multiple different maps (for different vehicle types) covering the same area, then layers can allow you to show/hide these different maps independently. You merely have to put them on different layers, then.

Best regards,
Stefan

**Reply by skyehan111** [reply]
Dear Stefan

Thank you for your answer, it is very helpful. I already have some ideas for using layers.

Thank you for your work.

Best Regards
Skye
